learning-basics/configure-yaml/src/main/java/ConfigLookup.java

```diff
@@ -0,0 +1,32 @@
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.config.Config;
+import com.hazelcast.config.YamlConfigBuilder;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+import static com.hazelcast.examples.helper.CommonUtils.sleepMillis;
+
+public class ConfigLookup {
+    public static void main(String[] args) {
+        // XML takes precedence: loading member configuration hazelcast.xml from the classpath
+        // despite the presence of hazelcast.yaml
+        HazelcastInstance member = Hazelcast.newHazelcastInstance();
+        sleepMillis(1000);
```

Comment:
Why do you need this sleeps? For the user to observe the results? Or this is necessary for some reason? Could you maybe comment it in the code?

Reply:
I used the `configuration-xml` sample as a basis, the sleeps were there. Removed, it wasn't necessary.